homeassistant/components/lupusec/alarm_control_panel.py

```diff
@@ -70,3 +71,13 @@ def alarm_disarm(self, code: str | None = None) -> None:
     def alarm_arm_home(self, code: str | None = None) -> None:
         """Send arm home command."""
         self._device.set_home()
+
+    @property
+    def device_info(self):
```

Comment:
This can be a shorthand attribute in the constructor

```py
self._attr_device_info = DeviceInfo
```

(Oh and please use the DeviceInfo TypedDict instead of a loose dict)

Reply:
Done.